### PR TITLE
Update the show-password command

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- GIJoe and see that command under the login screen is `--show-password`

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5466/
Test PASSed.

**Comment by huwshimi** [comment]
Looks fine, thanks @anthonydillon.

File: jujugui/static/gui/src/test/test_app.js [hunk 1]
```diff
@@ -693,7 +693,7 @@ describe('App', function() {
           assert.strictEqual(
             node.props.helpMessage,
             'Find your username and password with ' +
-            '`juju show-controller --include-passwords`'
+            '`juju show-controller --show-password`'
```
Comment (bac) [inline jujugui/static/gui/src/test/test_app.js]: How did this old test pass when it had `--include-passwords` and not the old `--show-passwords`?

**Comment by anthonydillon** [comment]
Thanks :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

File: jujugui/static/gui/src/test/test_app.js (hunk 1 above)
Comment (anthonydillon) [inline jujugui/static/gui/src/test/test_app.js]: I assume it was never run but thought I should update it anyway.

**Pull request merged**